nisystemlink/clients/systemsstate/_systems_state_client.py

```diff
@@ -0,0 +1,256 @@
+                how to connect. If not provided, the
+                :class:`HttpConfigurationManager <nisystemlink.clients.core.HttpConfigurationManager>`
+                is used to obtain the configuration.
+
+        Raises:
+            ApiException: if unable to communicate with the Spec Service.
+        """
+        if configuration is None:
+            configuration = core.HttpConfigurationManager.get_configuration()
+
+        super().__init__(configuration, base_path="/nisystemsstate/v1/")
+
+    @get(
+        "states",
+        args=[
+            Query("Skip"),
+            Query("Take"),
+            Query("Workspace"),
+            Query("Architecture"),
+            Query("Distribution"),
+        ],
+    )
+    def get_states(
+        self,
+        skip: Optional[int] = None,
+        take: Optional[int] = None,
+        workspace: Optional[str] = None,
+        architecture: Optional[models.Architecture] = None,
+        distribution: Optional[models.Distribution] = None,
+    ) -> models.StateDescriptionListResponse:
+        """Get the list of states
+
+        Args:
+            `skip`: Number of files to skip in the result when paging
+            `take`: Number of files to return in the result
+            `workspace`: The workspace id of the states
+            `architecture`: The states that are compatible with the requested architecture
+            `distribution`: The states that are compatible with the requested distribution
+
+        Returns:
+            The list of states for the given query parameters
+
+        Raises:
+            ApiException : if unable to communicate with the ``/nisystemsstate`` Service
+                or provided an invalid argument.
+        """
+        ...
+
+    @post("states", args=[Body])
+    def create_state(self, new_state: models.StateRequest) -> models.StateResponse:
+        """Create state
+
+        Args:
+            `state`: Information about the state to be created.
+
+        Returns:
+            The state created
+
+        Raises:
+            ApiException : if unable to communicate with the ``/nisystemsstate`` Service
+                or provided an invalid argument.
+        """
+        ...
+
+    @post("export-state", args=[Body])
+    def export_state(self, export_state_request: models.ExportStateRequest) -> str:
+        """Generate state export
+
+        Args:
+            `export_state_request`: Contains identifying information on the state to export.
+
+        Returns:
+            The exported state as a file.
+
+        Raises:
+            ApiException : if unable to communicate with the ``/nisystemsstate`` Service
+                or provided an invalid argument.
+        """
+        ...
+
+    @post("export-state-from-system", args=[Body])
+    def export_state_from_system(
+        self, export_state_from_system_request: models.ExportStateFromSystemRequest
+    ) -> str:
```

Comment:
```suggestion
    @params({"inline": True})  # type: ignore
    @post("export-state-from-system", args=[Field("systemID")])
    def export_state_from_system(self, system_id: str) -> str:
```
Eliminate the need for a request object by inlining parameters,
References for the recommendation,
https://github.com/ni/nisystemlink-clients-python/blob/caddc7501b240b8f4653447b7851f40f3c21de78/nisystemlink/clients/file/_file_client.py#L173
https://github.com/ni/nisystemlink-clients-python/blob/caddc7501b240b8f4653447b7851f40f3c21de78/nisystemlink/clients/product/_product_client.py#L31